net-fs/openafs/openafs-1.8.13.ebuild

```diff
@@ -0,0 +1,348 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MODULES_OPTIONAL_USE="modules"
+inherit autotools linux-mod flag-o-matic pam systemd tmpfiles toolchain-funcs
+
+MY_PV=${PV/_/}
+MY_P="${PN}-${MY_PV}"
+PVER=20211212
+KERNEL_LIMIT=6.11
+
+DESCRIPTION="The OpenAFS distributed file system"
+HOMEPAGE="https://www.openafs.org/"
+# We always d/l the doc tarball as man pages are not USE=doc material
+[[ ${PV} == *_pre* ]] && MY_PRE="candidate/" || MY_PRE=""
+SRC_URI="
+	https://openafs.org/dl/openafs/${MY_PRE}${MY_PV}/${MY_P}-src.tar.bz2
+	https://openafs.org/dl/openafs/${MY_PRE}${MY_PV}/${MY_P}-doc.tar.bz2
+"
+
+LICENSE="IBM BSD openafs-krb5-a APSL-2"
+SLOT="0"
+KEYWORDS="~amd64 -riscv ~sparc ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="apidoc bitmap-later debug doc fuse kauth kerberos +modules +namei
+ncurses perl +pthreaded-ubik selinux +supergroups tsm ubik-read-while-write"
+
+BDEPEND="
+	dev-lang/perl
+	app-alternatives/lex
+	app-alternatives/yacc
+	apidoc? (
+		app-text/doxygen[dot]
+		media-gfx/graphviz
+	)
+	doc? (
+		dev-libs/libxslt
+		|| (
+			dev-java/fop
+			app-text/dblatex
+			app-text/docbook-sgml-utils[jadetex]
+		)
+	)
+	perl? ( dev-lang/swig )"
+DEPEND="
+	!net-fs/openafs-kernel
+	virtual/libcrypt:=
+	virtual/libintl
+	amd64? ( tsm? ( app-backup/tsm ) )
+	doc? (
+		app-text/docbook-xsl-stylesheets
+		app-text/docbook-xml-dtd:4.3
+	)
+	fuse? ( sys-fs/fuse:0= )
+	kauth? ( sys-libs/pam )
+	kerberos? ( virtual/krb5 )
+	ncurses? ( sys-libs/ncurses:0= )"
+RDEPEND="
+	${DEPEND}
+	selinux? ( sec-policy/selinux-afs )"
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=(
+	"${FILESDIR}"/0001-autoconf-use-AC_CHECK_TOOL-for-as-and-ld.patch
+	"${FILESDIR}"/0003-fbsd.patch
+	"${FILESDIR}"/0005-uname.patch
+	"${FILESDIR}"/0007-afsauthent-symbols.patch
+	"${FILESDIR}"/0009-docbook2pdf.patch
+	"${FILESDIR}"/0011-xbsa.patch
+	"${FILESDIR}"/0013-kernel-cc-ld.patch
+    "${FILESDIR}"/0002-pam-paths.patch
```

Comment:
These lines are inconsistently indented. Also, we're not applying in the same order as the numbering..?

Reply:
I sorted these, but didn't notice the mix of prefixed spaces and tabs which threw off the sort.